config/v1/types_cluster_version.go

```diff
@@ -105,13 +106,15 @@ type ClusterVersionStatus struct {
 	// by a temporary or permanent error. Conditions are only valid for the
 	// current desiredUpdate when metadata.generation is equal to
 	// status.generation.
-	Conditions []ClusterOperatorStatusCondition `json:"conditions"`
+	// +optional
+	Conditions []ClusterOperatorStatusCondition `json:"conditions,omitempty"`
 
 	// availableUpdates contains the list of updates that are appropriate
 	// for this cluster. This list may be empty if no updates are recommended,
 	// if the update service is unavailable, or if an invalid channel has
 	// been specified.
-	AvailableUpdates []Update `json:"availableUpdates"`
+	// +optional
+	AvailableUpdates []Update `json:"availableUpdates,omitempty"`
```

Comment:
@sttts I prefer optional and not omitempty.

@abhinavdahiya @smarterclayton we're fixing tags to be able to generate CRD schemas for both CRD validation and `oc explain`.  Check yours

Reply:
not omitempty means that we get a `null` potentially. @deads2k do you want that?

Reply:
it's now nullable